test/cli/autogen-multipkg-autoloader/foo/__package.rb

```diff
@@ -3,5 +3,5 @@
 class Project::Foo < PackageSpec
   import Project::Bar
 
-  export Foo
+  export Project::Foo::FooClass
```

Comment:
Why are the full prefixes required in the `export` declaration now?

Reply:
Do these still end up needing to be fully-qualified? Would it be easy to make them scoped as though they were defined under `Project::Foo`?

Reply:
It would be easy to go either way. My thought here was to force the FQ name for grep-ability

Reply:
Is this easy to change if people complain in the future?